src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -208,17 +208,22 @@ private void writeSAMfilesForUnknown(final String outputPrefix, final JavaRDD<GA
 
             final List<GATKRead> contigRawAlignments = assemblyRawAlignments
                     .filter(read -> namesOfInterest.contains(read.getName())).collect();
+            contigRawAlignments.sort(Comparator.comparing(GATKRead::getName));
```

Comment:
Do we always want to write these files sorted by name even though the other bam file's sort order is now choosable from the command line? You might want to put a note in the documentation for the new parameter that it will not affect these BAM files.

Reply:
Yes. Will do.

Users most likely will not care about these bams, because they are more of a debugging use. 

These are local assembly contigs whose alignment signatures that we don't know how to emit un-ambiguous calls from, so their alignments are saved just incase we want to emit BND's, or which ever format we can report, and annotate accordingly.

Personally I generally look at all alignments of a contig to figure out why they are classified that way, and get a sense of if we can come up with rules to squeeze more signal from them.

I also tried with an option of NOT outputting alignments of these contigs, but just names and the reason for failing to produce a call. But in the end I find myself using `ExtractOriginalAlignmentRecordsByNameSpark` again to extract the alignments anyway, so just decided to have some data redundancy in exchange for convenience.